### PR TITLE
Add image placeholder to Psammead

### PR DESCRIPTION
**Pull request by radiocontrolled**
Resolves https://github.com/BBC-News/psammead/issues/211

Adds an image placeholder to Psammead.

**Screenshots**
screen shot 2018-12-17 at 10 17 23

screen shot 2018-12-17 at 10 17 34

screen shot 2018-12-17 at 10 17 41

- [x] I have assigned myself to this PR and the corresponding issues
- [N/A] Tests added for new features - existing tests pass
- [ ] Test engineer approval

### CODE REVIEW

**Review by pjlee11 — DISMISSED** [review]

File: packages/components/psammead-image-placeholder/README.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# psammead-image-placeholder &middot; [![GitHub license](https://img.shields.io/badge/license-Apache%202.0-blue.svg)](https://github.com/BBC-News/psammead/blob/latest/LICENSE) [![npm version](https://img.shields.io/npm/v/@bbc/psammead-image-placeholder.svg)](https://www.npmjs.com/package/@bbc/psammead-image-placeholder) [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg)](https://github.com/BBC-News/psammead/blob/latest/CONTRIBUTING.md)
+
+## Description
+
+Psammead Image Placeholder is a `div` with a base-64 encoded SVG background image that depicts the BBC logo. The component accepts a ratio as a prop. In the Psammead component library, Image Placeholder is used within the [psammead-figure](https://github.com/BBC-News/psammead/tree/latest/packages/components/psammead-figure) component.
```
Comment (ChrisBAshton) [inline packages/components/psammead-image-placeholder/README.md:5]: This is not a thing yet - but we have raised https://github.com/BBC-News/psammead/issues/214.

Agreed we should get this PR through before tackling Figure.

File: packages/components/psammead-image-placeholder/src/index.stories.jsx [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+const landscapeImageRatio = 56.25;
+const squareImageRatio = 100;
+const portraitImageRatio = 177.78;
+
+storiesOf('ImagePlaceholder', module)
+  .add('16x9 image placeholder', () => (
+    <ImagePlaceholder ratio={landscapeImageRatio} />
+  ))
+  .add('1x1·image·placeholder', () => (
```
Comment (ChrisBAshton) [inline packages/components/psammead-image-placeholder/src/index.stories.jsx]:
```suggestion
  .add('1x1 image placeholder', () => (
```

File: packages/components/psammead-image-placeholder/src/index.test.jsx [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+describe('ImagePlaceholder', () => {
+  shouldMatchSnapshot(
+    'should render landscape images correctly',
+    <ImagePlaceholder ratio={landscapeImageRatio} />,
+  );
+  shouldMatchSnapshot(
+    'should render portrait images correctly',
+    <ImagePlaceholder ratio={portraitImageRatio} />,
+  );
```
Comment (ChrisBAshton) [inline packages/components/psammead-image-placeholder/src/index.test.jsx:17]: Should we also do the 1x1 test here?

File: packages/components/psammead-image-placeholder/src/index.test.jsx [hunk 4]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { shouldMatchSnapshot } from '@bbc/psammead-test-helpers';
+import ImagePlaceholder from './index';
```
Comment (ChrisBAshton) [inline packages/components/psammead-image-placeholder/src/index.test.jsx]:
```suggestion
import ImagePlaceholder from '.';
```

File: packages/components/psammead-image-placeholder/src/index.stories.jsx [hunk 5]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react'; // eslint-disable-line import/no-extraneous-dependencies
+import ImagePlaceholder from './index';
```
Comment (ChrisBAshton) [inline packages/components/psammead-image-placeholder/src/index.stories.jsx]:
```suggestion
import ImagePlaceholder from '.';
```

**Review by ChrisBAshton — COMMENTED** [review]
Good job!

File: packages/components/psammead-image-placeholder/src/index.test.jsx (hunk 3 above)
Comment (radiocontrolled) [reply packages/components/psammead-image-placeholder/src/index.test.jsx:17]: Ok I'll update it.

File: packages/components/psammead-image-placeholder/src/index.stories.jsx [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+
+const landscapeImageRatio = 56.25;
+const squareImageRatio = 100;
+const portraitImageRatio = 177.78;
+
+storiesOf('ImagePlaceholder', module)
+  .add('16x9 image placeholder', () => (
+    <ImagePlaceholder ratio={landscapeImageRatio} />
+  ))
+  .add('1x1 image·placeholder', () => (
```
Comment (ChrisBAshton) [inline packages/components/psammead-image-placeholder/src/index.stories.jsx]:
```suggestion
  .add('1x1 image placeholder', () => (
```

**Review by pjlee11 — DISMISSED** [review]

File: packages/components/psammead-image-placeholder/package.json [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+  },
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/bbc/psammead/issues"
+  },
+  "homepage": "https://github.com/BBC-News/psammead/blob/latest/packages/components/psammead-image-placeholder/README.md",
+  "dependencies": {
+    "@bbc/psammead-assets": "^0.1.2",
+    "@bbc/psammead-styles": "^0.1.3",
+    "react": "^16.6.3",
```
Comment (dr3) [inline packages/components/psammead-image-placeholder/package.json]: React is a dev dependancy, not a normal one

**Review by dr3 — CHANGES_REQUESTED** [review]

File: packages/components/psammead-image-placeholder/package.json (hunk 7 above)
Comment (ChrisBAshton) [reply packages/components/psammead-image-placeholder/package.json]: True - the "real" dependency here is styled-components

Comment (radiocontrolled) [reply packages/components/psammead-image-placeholder/package.json]: @dr3 your comment is addressed.

**Review by ChrisBAshton — APPROVED** [review]

**Review by dr3 — APPROVED** [review]

**Comment by jamesbrumpton** [comment]
LGTM

**Pull request merged**